src/main/java/com/manardenza/dao/UserDaoImpl.java

```diff
@@ -2,10 +2,17 @@
 
 import com.manardenza.entity.User;
 
+import java.io.File;
+import java.util.List;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.List'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)